kafka_consumer/tests/python_client/test_unit.py

```diff
@@ -25,10 +24,25 @@ def test_gssapi(kafka_instance, dd_run_check, check):
     instance['sasl_kerberos_service_name'] = 'kafka'
     # assert the check doesn't fail with:
     # Exception: Could not find main GSSAPI shared library.
-    with pytest.raises(Exception, match='check_version'):
+    with pytest.raises(Exception):
         dd_run_check(check(instance))
 
 
+def test_gssapi_config(kafka_instance, dd_run_check, check, caplog):
+    instance = copy.deepcopy(kafka_instance)
+    instance['sasl_mechanism'] = 'GSSAPI'
+    instance['security_protocol'] = 'SASL_PLAINTEXT'
+    instance['sasl_kerberos_service_name'] = 'kafka'
+    instance['sasl_kerberos_domain_name'] = "Not none"
+
+    # TODO: Mock GSSAPI here instead of expecting fail
+    with pytest.raises(Exception):
+        dd_run_check(check(instance))
```

Comment:
To reviewer: I left this as a TODO for now because the Kerberos tests should be refactored together as a whole in a separate PR to test the config and to test the actual functionality with the config.